osconfig_tests/test_suites/package_management/package_management_test_data.go

```diff
@@ -68,24 +76,35 @@ func addPackageInstallTest(pkgTestSetup []*packageManagementTestSetup) []*packag
 	for _, pkgManager := range pkgManagers {
 		var oc *osconfigpb.OsConfig
 		var assign *osconfigpb.Assignment
-		var instaneName, ss, vs string
+		var instanceName, image, ss, vs string
 
 		switch pkgManager {
 		case "apt":
-			instaneName = fmt.Sprintf("%s-%s", filepath.Base(debianImage), testName)
+			image = debianImage
+			instanceName = fmt.Sprintf("%s-%s", filepath.Base(image), testName)
 			pkg := osconfigserver.BuildPackage("cowsay")
 			pkgs := []*osconfigpb.Package{pkg}
 			oc = osconfigserver.BuildOsConfig(testName, desc, osconfigserver.BuildAptPackageConfig(pkgs, nil, nil), nil, nil, nil, nil)
-			assign = osconfigserver.BuildAssignment(testName, desc, osconfigserver.BuildInstanceFilterExpression(instaneName), []string{fmt.Sprintf("projects/%s/osConfigs/%s", testProjectID, oc.Name)})
+			assign = osconfigserver.BuildAssignment(testName, desc, osconfigserver.BuildInstanceFilterExpression(instanceName), []string{fmt.Sprintf("projects/%s/osConfigs/%s", testProjectID, oc.Name)})
 			ss = fmt.Sprintf("%s\nwhile true; do /usr/bin/dpkg-query -s cowsay | sudo tee /dev/ttyS0; sleep 5; done", utils.InstallOSConfigDeb)
 			vs = fmt.Sprintf("install ok installed")
 			break
+		case "yum":
+			image = centosImage
+			instanceName = fmt.Sprintf("%s-%s", filepath.Base(image), testName)
+			pkg := osconfigserver.BuildPackage("cowsay")
+			pkgs := []*osconfigpb.Package{pkg}
+			oc = osconfigserver.BuildOsConfig(testName, desc, nil, osconfigserver.BuildYumPackageConfig(pkgs, nil, nil), nil, nil, nil)
+			assign = osconfigserver.BuildAssignment(testName, desc, osconfigserver.BuildInstanceFilterExpression(instanceName), []string{fmt.Sprintf("projects/%s/osConfigs/%s", testProjectID, oc.Name)})
+			ss = fmt.Sprintf("%s\nwhile true; do /usr/bin/rpmquery -a cowsay | sudo tee /dev/ttyS0; sleep 5; done", utils.InstallOSConfigYumEL7)
+			vs = fmt.Sprintf("cowsay")
```

Comment:
pointless Sprintf, same on line 90, and elsewhere

Reply:
yeah... this should have been cleaned up. anyways, this is now moved and should be much cleaner now.